src/api/Feed.js

```diff
@@ -1220,6 +1274,53 @@ class Feed extends Base {
         this.feedErrorCallback(false, e, code);
     };
 
+    /**
+     * Callback for successful creation of a Comment.
+     *
+     * @param {Comment} commentData - API returned Comment
+     * @param {string} parentId - ID of the parent feed item
+     * @param {string} id - ID of the reply to update with the new comment data
+     * @param {Function} successCallback - success callback
+     * @return {void}
+     */
+    createReplySuccessCallback = (
+        commentData: Comment,
+        parentId: string,
+        id: string,
+        successCallback: Function,
+    ): void => {
+        this.updateReplyItem(
+            {
+                ...commentData,
+                isPending: false,
+            },
+            parentId,
+            id,
+        );
+
+        if (!this.isDestroyed()) {
+            successCallback(commentData);
+        }
+    };
+
+    /**
+     * Callback for failed creation of a reply.
+     *
+     * @param {Object} e - The axios error
```

Comment:
should this be updated to `error`?

Reply:
Should the param type be `ElementsXhrError`? Or does `Object` suffice??

Reply:
That's correct. I've updated JSDoc so that it's aligned with Flow.